packages/web/docs/src/components/product-updates.tsx

```diff
@@ -30,57 +42,82 @@ export function ProductUpdateTeaser(props: Changelog): ReactElement {
   );
 }
 
+// TODO: I'm not sure this is gonna work. We previously imported Nextra internal here.
 export async function getChangelogs(): Promise<Changelog[]> {
-  const { pageMap } = await import('../../.next/static/chunks/nextra-page-map-.mjs');
+  const [_meta, ...pageMap] = await getPageMap();
```

Comment:
```suggestion
  const [_meta, ...pageMap] = await getPageMap('/product-updates');
```
and you'll get specific `pageMap` based on your route (default `/`)

Reply:
nice